FhirToDataLake/azurepipelines/ci.yml

```diff
@@ -0,0 +1,106 @@
+steps:
+- task: NuGetAuthenticate@0
+  displayName: 'NuGet Authenticate'
+
+- task: NuGetToolInstaller@1
+  displayName: 'Use NuGet 5.11'
+  inputs:
+    versionSpec: 5.11
+
+# Install for Azure Function dependencies
+- task: UseDotNet@2
+  displayName: 'Install .NET Core sdk 3.1'
+  inputs:
+    packageType: sdk
+    version: 3.1.402
+    installationPath: $(Agent.ToolsDirectory)/dotnet
+
+# Install for Azure Function dependencies
+- task: UseDotNet@2
+  displayName: 'Use .NET Core sdk 6.0'
+  inputs:
+    packageType: sdk
+    version: 6.0.302
+    installationPath: $(Agent.ToolsDirectory)/dotnet
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet restore solution'
+  inputs:
+    command: restore
+    projects: FhirToDataLake/*.sln
+    feedsToUse: 'config'
+    nugetConfigPath: 'FhirToDataLake/NuGet.Config'
+
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet build solution'
+  inputs:
+    command: build
+    projects: FhirToDataLake/*.sln
+    arguments: '--configuration $(buildConfiguration) -p:Version=$(version)'
+
+- script: |
+    docker pull mcr.microsoft.com/azure-storage/azurite
+    docker run -d --name azurite-emulator-analytics-ci -p 10000:10000 -p 10001:10001 -p 10002:10002 \
```

Comment:
Maybe using `-p 10000-10002:10000:10002` for simplification.

Reply:
Updated. Thanks